tests/integration/test_yarn.py

```diff
@@ -65,6 +65,20 @@
             ),
             id="yarn_immutable_installs",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-yarn-berry.git",
+                ref="dcb16b5931a89ac642be2dcf9886e148a905bea4",
```

Comment:
nit: Consider updating this ref to the latest on that branch so that it links to the updated README

Reply:
:facepalm: